testgrid/metadata/job.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+// Started holds the started.json values of the build.
+type Started struct {
```

Comment:
This should embed a metadata?

Reply:
Done

Reply:
The thought for things that are k8s-specific especially k/k-bash-specific they can sit in Metadata unless otherwise specified? If we inline the metadata struct we're not changing the location of the keys. Is that possible? Fields like `node` `pull` `repo-version` and `repos` are specific to bootstrap and/or k/k

Reply:
Sure, I would like to keep all these fields now because they are in use.

Can we discuss how to move the format ahead in future PRs?

I want to be able to use this to process our existing results